mlora.py

```diff
@@ -42,6 +42,8 @@
                     help="Disable the adapter modules")
 parser.add_argument('--tokenizer', type=str,
                     help='Path to or name of tokenizer')
+parser.add_argument('--flash_attn', action='store_true',
```

Comment:
from "clean code" by Robert C. Martin, it's suggest don't use flag argument. split or use type to replace it.

Reply:
How about changing to `--attn_impl auto|eager|xformers|flash_attn`?

Reply:
use attn_impl maybe clear

Reply:
GOOD